bazel/repo/clang_tidy_wrapper.sh.tpl

```diff
@@ -2,7 +2,23 @@
 
 set -e
 
+# HACK(chenhao94): Skip all the "textual" sources.
+if printf "%s\n" "$@" | grep "\.txt\.h\(pp\)\?$" > /dev/null; then
+    exit 0;
+fi
+
 # HACK(chenhao94): Bazel will redefine built-in macros (e.g., "__DATE__") for
 # deterministic builds, and it triggers the Clang Tidy check.
 # See https://github.com/erenon/bazel_clang_tidy/issues/29
-%{CLANG_TIDY} --checks=-clang-diagnostic-builtin-macro-redefined "$@"
+# HACK(chenhao94): Skip all the generated protobuf headers.
+%{CLANG_TIDY} \
+    --checks=-clang-diagnostic-builtin-macro-redefined \
+    --line-filter='[
+            {"name":".pb.h","lines":[[9999999,9999999]]},
```

Comment:
What does the "lines" means here, and why 9999999?

Reply:
lines means line ranges that need to be checked. "9999999" means to skip the file, since there are no such line numbers in this file.

Reply:
I think the correct way should be putting the filter rules in the "Header Filter Regex" in `.clang-tidy`. But I don't know how to exclude files or maybe we use different prefix for auto-generated headers.

Reply:
Sounds good.